lms/static/sass/base/_variables.scss

```diff
@@ -82,6 +82,27 @@ $gray-d2: shade($gray,40%); // #4c4c4c
 $gray-d3: shade($gray,60%); // #323232
 $gray-d4: shade($gray,80%); // #191919
 
+$cms-blue: rgb(0, 159, 230); // NOTE - eventually, this set of blues should be synced with CMS.  
```

Comment:
I went ahead and backed out these blues to use the m-blue-* colors, and added back in the older blue-d1; etc used by other parts of the LMS.